lib/LaTeXML/Package/etoolbox.sty.ltxml

```diff
@@ -1296,15 +1296,15 @@ DefMacro('\patchcmd [] DefToken {}{}{}{}', sub {
         Info('unexpected', 'patchcmd', $gullet, "Patchcmd is not supported on LaTeXML-native definitions, will not patch " . ToString($cs));
         return $failure;
       }
-      my $string        = ToString($expansion);
-      my $search_string = ToString($search);
+      my $string        = UnTeX($expansion);
+      my $search_string = UnTeX($search);
```

Comment:
You'll probably want to add the optional `$suppress_linebreak` argument here, (ie. `,1`) just in case.

Reply:
Oh, sure!

Reply:
Thanks for beating me to it.